python/paddle/v2/framework/tests/gradient_checker.py

```diff
@@ -99,6 +104,10 @@ def product(dim):
         y_pos = get_output()
 
         # plus delta to this element, run op and get the sum of the result tensor.
+        if in_place:
```

Comment:
may be should move this code to a function
```python
def restore_inputs(input_values, scope):
        for var_name in input_values:
             tensor_ = scope.find_var(var_name).get_tensor()
             tensor_.set(numpy.copy(input_values[var_name]), core.CPUPlace())
```

Reply:
moved into copy_tensor()

Reply:
I think copy_tensor is not a very good name. Can not know what it means by reading the name. maybe `restore_inputs` or `reset_inputs`